src/main/java/net/preibisch/mvrecon/fiji/plugin/interestpointdetection/interactive/InteractiveDoG.java

```diff
@@ -296,8 +296,8 @@ protected void updatePreview(final ValueChange change) {
 		}
 
 		final double radius = ( ( params.sigma + HelperFunctions.computeSigma2( params.sigma, sensitivity ) ) / 2.0 );
-		final ArrayList< RefinedPeak< Point > > filteredPeaksMax = HelperFunctions.filterPeaks( peaksMax, rectangle, params.threshold );
-		final ArrayList< RefinedPeak< Point > > filteredPeaksMin = HelperFunctions.filterPeaks( peaksMin, rectangle, params.threshold );
+		final ArrayList< RefinedPeak< Point > > filteredPeaksMax = HelperFunctions.filterPeaks( peaksMax, rectangle, params.threshold / 2.5 ); // correction factor of 2.5 applied to match thresholding in final DoG IP detection
```

Comment:
I am wondering why this makes sense (I assume you tested it and it does make sense) ... will test this as well and merge for now, thanks so much!